### DISCUSSION TITLE
slack invite

### DISCUSSION THREAD

**Discussion by pefeigl** [fluxcd/flux2#2746] [General] [answered]
I've not been able to find a working slack invite, they all seem to be expired. Could you possible update them?

**Comment by dholbach** [comment, marked as answer]
I told the CNCF infrastructure team - I hope they will fix it soon. Will let you know when we have an update.

**Reply by dholbach** [reply]
They responded and acknowledged the problem and will work on resolving it.

**Reply by dholbach** [reply]
Apparently it's fixed now, but might take a few minutes for DNS to propagate. So hang in there for a little longer!

**Reply by mprusinski** [reply]
Hi, the same goes for me - I have tried to use invite link from GitHub Readme file and it just says that the link is no longer active.

**Reply by dholbach** [reply]
Can you try https://slack.cncf.io again - maybe in a private/incognito browser window?
